File: 88-MergeSortedArray/88-MergeSortedArray.java
// Last updated: 05/06/2026, 23:41:14
1class Solution {
2    public void merge(int[] n1, int m, int[] n2, int n) {
3        for(int i=m,j=0;i<n1.length;i++)
4        {
5            n1[i]=n2[j];
6            j++;
7        }
8        Arrays.sort(n1);
9    }
10}

File: 88-MergeSortedArray/88-MergeSortedArray.py
# Last updated: 05/08/2026, 10:32:02
class Solution(object):
    def merge(self, nums1, m, nums2, n):
        j=0
        for i in range(m,m+n):
            nums1[i]=nums2[j]
            j+=1
        nums1.sort()
        return nums1
        